packages/vbsmin/PKGBUILD

```diff
@@ -0,0 +1,52 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=vbsmin
+pkgver=v1.0.0.r1.g1f6790d
+pkgrel=1
+groups=('blackarch' 'blackarch-automation' 'blackarch-packer')
+pkgdesc='VBScript minifier.'
+arch=('any')
+url='https://noraj.github.io/vbsmin/'
+license=('MIT')
+depends=('ruby' 'ruby-paint' 'ruby-docopt')
+makedepends=('git')
+options=(!emptydirs)
+source=("git+https://github.com/noraj/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  git describe --long | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
+}
+
+# waiting for ruby-paint 2.2.0 to be released on AL repo
+prepare() {
+  cd $pkgname
+
+  sed -i "s/'paint', '~> 2.2'/'paint', '>= 2.1'/" "$pkgname.gemspec"
```

Comment:
Use `sed -i 's/~> 2.2/>= 2.1/g'`

Reply:
Shorter but less future proof and anyway will be remove when https://www.archlinux.org/packages/community/any/ruby-paint/ will be updated

Reply:
Don't open PRs then if you're going to omit comments, push directly to master.